src/traces/sunburst/plot.js

```diff
@@ -487,10 +487,31 @@ exports.formatSliceLabel = function(pt, entry, trace, cd, fullLayout) {
     var cd0 = cd[0];
     var cdi = pt.data.data;
     var hierarchy = cd0.hierarchy;
+    var isRoot = helpers.isHierarchyRoot(pt);
+    var isEntry = helpers.isEntry(pt);
+    var parent = isEntry ? pt._parent : pt.parent;
+
     var ref;
+    var getLabel = function(d) {
+        if(d.hasOwnProperty('hierarchy')) return d.hierarchy.label;
+        if(!(d.data && d.data.data)) return '';
+        var id = helpers.getPtId(d);
+        for(var q = 0; q < cd.length; q++) {
+            if(cd[q].label === id) {
+                return id;
+            }
+        }
+        return '';
+    };
+
+    var getVal = function(d) {
+        if(d.hasOwnProperty('hierarchy')) return d.hierarchy.value;
```

Comment:
Where does `d` get a `hierarchy` key?

Reply:
Here: https://github.com/plotly/plotly.js/blob/217c0f43020f9f13d4361aadc6cd2d1a93c020ae/src/traces/treemap/plot.js#L90-L93

Reply:
Removed in https://github.com/plotly/plotly.js/pull/4185/commits/8416883be97b77bdf685ade8790544e4d3b0fa8a.